### ISSUE TITLE
Cannot install on Docker 17.03 on Mac

### ISSUE THREAD

**Issue by lkrzyzanek** [redhatdemocentral/ocp-install-demo#24]
Installer requires docker 1.3
I have on mac official docker 17.03

Terminal says:
`Docker engine version 17.03 found... need 1.13, please install: https://drive.google.com/open?id=0B9WSViV5BZ4aVXV5U3F4LVVmWVk`

**Comment by eschabell** [comment]
If you follow the link provided "https://drive.google.com/open?id=0B9WSViV5BZ4aVXV5U3F4LVVmWVk" you will find docker installer for 1.13, install that (next to or over existing version) but DON'T UPGRADE when offered, that keeps you on 1.13.

Are the instructions in the terminal unclear? How can I improve them?

**Comment by lkrzyzanek** [comment]
Instructions are clear.
I saw that install DMG file on google drive but I don't want to install another version of docker. I get rid of it couple months ago and I'd like to keep just one docker.
Even more I don't feel very safety to install some application from some google drive.

Is it necessary to require specific version of docker?

**Comment by lkrzyzanek** [comment]
To be more precise this is output of current version:
```
$ docker version
Client:
 Version:      17.03.0-ce
 API version:  1.26
 Go version:   go1.7.5
 Git commit:   60ccb22
 Built:        Thu Feb 23 10:40:59 2017
 OS/Arch:      darwin/amd64

Server:
 Version:      17.03.0-ce
 API version:  1.26 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   3a232c8
 Built:        Tue Feb 28 07:52:04 2017
 OS/Arch:      linux/amd64
 Experimental: true
```

**Comment by eschabell** [comment]
OCP does not support this version 17.x. I have to host this on google drive for my workshop(s) as it is constantly disappearing or changing locations on their site.

You can safely host two docker installs next to each other, just be sure to start the 1.13 if you want to run this demo installation. 

I will update this URL to point to https://download.docker.com/mac/stable/1.13.1.15353/Docker.dmg as this is the last place I can find it....

**Comment by lkrzyzanek** [comment]
Played with Docker 17 on Mac and here are steps to do:

1. Add `172.30.0.0/16` to insecure registry in docker preferences
2. Make sure you have installed `socat`. On mac you can install via brew e.g. `brew install socat`
3. oc cluster up --image=registry.access.redhat.com/openshift3/ose --version=v3.5

In a while OCP is available on `https://127.0.0.1:8443`.

That's it.

**Comment by eschabell** [comment]
Thanks for that input, it might work for getting OCP up, but it causes problems with container builds using projects from this organization. I will have to double check that.

**Comment by eschabell** [comment]
Testing more appdev in the cloud demo projects on top, but looks like this works now with 17.03 as I nail down docker 1.13 for the cluster up options to ensure internally it's using v1.13. I don't need the 'scout' install here... not sure why yours does? Might be your cluster up or oc client is missing something that mine does not need (it's not installed on my machine at all.

I will be committing an update soon to support the latest local docker. :-)

**Comment by lkrzyzanek** [comment]
The reason for `socat` is this:
````
oc cluster up --image=registry.access.redhat.com/openshift3/ose --version=v3.5
-- Checking OpenShift client ... OK
-- Checking Docker client ... OK
-- Checking Docker version ...
   WARNING: Cannot verify Docker version
-- Checking for existing OpenShift container ... OK
-- Checking for registry.access.redhat.com/openshift3/ose:v3.5 image ... OK
-- Checking Docker daemon configuration ... OK
-- Checking for available ports ... OK
-- Checking type of volume mount ...
   Using Docker shared volumes for OpenShift volumes
-- Creating host directories ... OK
-- Finding server IP ...
   Using 127.0.0.1 as the server IP
-- Starting OpenShift container ... FAIL
   Error: exec: "socat": executable file not found in $PATH
````

Thanks for the update to the latest docker.

I have 17.03.0-ce-mac2 and during spinning up some testing EAP7 I had some troubles because OC was not able to work with such "long version" but it should be fixed in OC 3.6.

**Comment by eschabell** [comment]
The current init.sh is not falling over this Error... I don't have socat installed...
